post-scan-actions/aws-python-conformity-custom-check/handler.py

```diff
@@ -52,8 +52,8 @@ def lambda_handler(event, context):
             ccaccountid = get_cc_accountid(account_id)
 
             # Get the bucket & object details
-            bucketname = message["file_url"].split(".s3.amazonaws.com/")[0][8:]
-            fileprefix = message["file_url"].split(".s3.amazonaws.com/")[1]
+            bucketname = message["file_url"].split(".s3.")[0][8:]
+            fileprefix = message["file_url"].split(".amazonaws.com/")[1]
             filename = message["file_url"].split("/")[-1]
```

Comment:
Consider what we've been using in post-scan-action-tag.
```javascript
import urllib.parse

s3_domain_pattern = 's3(\..+)?\.amazonaws.com'
url = urllib.parse.urlparse(message['file_url'])
# check pre-signed URL type, path or virtual
if re.fullmatch(s3_domain_pattern, url.netloc):
    bucket = url.path.split('/')[1]
    s3_object = '/'.join(url.path.split('/')[2:])
else:
    bucket = url.netloc.split('.')[0]
    s3_object = url.path[1:]

object_key = urllib.parse.unquote_plus(s3_object)
```

Reply:
or https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/post-scan-actions/aws-python-promote-or-quarantine/handler.py#L50-62

Reply:
Updating now, will push changes through later tonight